scout-hellojs-app/src/main/resources/archetype-resources/__rootArtifactId__.app/src/main/java/app/AppServletContributors.java

```diff
@@ -1,3 +1,12 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
```

Comment:
Other archetype files seems not to have this header.

Reply:
Removed it